test/integration/helpers.js

```diff
@@ -22,19 +23,16 @@ exports.login = function(loginPageAlreadyLoaded) {
   // The login page doesn't use angular, so we have to use the underlying WebDriver instance
   var driver = browser.driver;
   if (!loginPageAlreadyLoaded) {
-    browser.get('/');
+    driver.get('/');
```

Comment:
@benjaminapetersen Object.keys won't find properties in the prototype chain, which might explain it